integration/src/integration/java/pl/allegro/tech/hermes/integration/PublishingTest.java

```diff
@@ -113,27 +113,6 @@ public void shouldNotConsumeMessagesWhenSubscriptionIsSuspended() {
     }
 
     @Test
-    public void shouldSendPendingMessagesAfterSubscriptionIsResumed() {
```

Comment:
I presume this functionality still holds but there is something odd with the behaviour of the test now?